compose/cli/command.py

```diff
@@ -107,10 +107,18 @@ def get_config_path(self, file_path=None):
             'fig.yaml',
         ]
 
-        def expand(filename):
-            return os.path.join(self.base_dir, filename)
+        def search_candidates(path):
+            candidates = [filename for filename in supported_filenames
+                          if os.path.exists(os.path.join(path, filename))]
 
-        candidates = [filename for filename in supported_filenames if os.path.exists(expand(filename))]
+            if len(candidates) == 0:
```

Comment:
I've opted for consistency with previous code of @aanand: see [L115 of parent commit](https://github.com/docker/compose/pull/1232/files#diff-a04be36073678de95917341f90c2cf58L115). But thanks for comment :)